help.rst

```diff
@@ -0,0 +1,170 @@
+Help
+====
+
+There are many ways to get help with your scientific computing and
+data needs - in fact, so many you don't know what to use.  This page
+lists who is available to help you, and ways of contacting them.
+
+
+Formulate your question
```

Comment:
Comment: I would move this at the end and in parallel work on gitlab template for issue tracker.

Reply:
I get so many questions that lack these basics, that I wanted to emphasize this as much as possible, so that it couldn't be missed.  That's why it's at the top.  I wouldn't mind too much moving it down, but perhaps we can make it shorter here or better introduce it so it flows better?

Reply:
yeah I know what you mean. We can leave it like that. Maybe it's good to ping @mtwest2718 if he can suggest improvements to the page (ping also to @essihe and @hastinarimanzadeh): you three have been at Aalto the least so you have a better perspective from an "external" point of view to improve this page.